### PR TITLE
TW-808: renamed application name in push notification messages

### PR DESCRIPTION
**Pull request by imGok**
kept only lang I'm sure it's correct : de,en,fr,vn,ru 

# Updated localization for iOS Notification

Screenshot 2023-10-17 at 11 16 07

### CODE REVIEW

File: assets/l10n/intl_ca.arb [hunk 1]
```diff
@@ -916,16 +916,6 @@
...
-    "newMessageInFluffyChat": "Missatge nou al FluffyChat",
-    "@newMessageInFluffyChat": {
-        "type": "text",
-        "placeholders": {}
-    },
-    "newVerificationRequest": "Nova sol·licitud de verificació!",
-    "@newVerificationRequest": {
-        "type": "text",
-        "placeholders": {}
-    },
```
Comment (hoangdat) [inline assets/l10n/intl_ca.arb:928]: why need to delete it?

**Comment by drminh2807** [comment]
I think for consistency between messages
we also need update "SINGLE_UNREAD" = "1 unread chat(s)";

**Comment by hoangdat** [comment]
I will ask BA for more details. Please wait

File: assets/l10n/intl_ca.arb (hunk 1 above)
Comment (imGok) [reply assets/l10n/intl_ca.arb]: missclicked

**Comment by imGok** [comment]
> SINGLE_UNREAD

it's a notification only for iOS right ?

**Comment by hoangdat** [comment]
"You have 1 encrypted message"

**Review by drminh2807 — APPROVED** [review]

File: ios/en.lproj/Localizable.strings [hunk 2]
```diff
@@ -5,4 +5,4 @@
   Created by Minh DrMinh on 7/19/23.
   
 */
-"SINGLE_UNREAD" = "1 unread chat(s)";
+"SINGLE_UNREAD" = "You have 1 encrypted message";
```
Comment (drminh2807) [inline ios/en.lproj/Localizable.strings:8]: you also update or add SINGLE_UNREAD in `vi.lproj/Localizable.strings` and `fr.lproj/Localizable.strings`

**Comment by imGok** [comment]
waiting for a xcode file from minh

**Comment by drminh2807** [comment]
> waiting for a xcode file from minh

done

**Review by hoangdat — APPROVED** [review]

**Pull request merged**